Composer/packages/client/src/components/CreateSkillModal.tsx

```diff
@@ -83,7 +83,9 @@ export const validateManifestUrl = async ({
   } else {
     try {
       setValidationState({ ...validationState, manifestUrl: ValidationState.Validating });
-      const { data } = await httpClient.post(`/projects/${projectId}/skill/check`, { url: manifestUrl });
+      const { data } = await httpClient.post(`/projects/${projectId}/skill/retrieve-skill-manifest`, {
```

Comment:
nitpick. Feel free to ignore if a lot of work.

The endpoint is a Post. It's name is retrieve-skill-manifest. Ideally this would be GET with the query param being the url encoded manifest URL.

Reply:
Yea, I thought about it that lol..